### PR TITLE
introduce `run_v21`

### PR DESCRIPTION
**Pull request by seuros**

### CODE REVIEW

File: lib/trailblazer/rails/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Trailblazer
   module Rails
-    VERSION = "2.1.4"
+    VERSION = "2.1.5"
```
Comment (emaglio) [inline lib/trailblazer/rails/version.rb]: I think this change together with Changelog should be on a different commit "Release 2.1.5"

Comment (seuros) [reply lib/trailblazer/rails/version.rb]: We release once this change is merged.

**Comment by seuros** [comment]
Yes. in the next major version you can update from 2.1 to 2.2.  
trailblazer-future will require trailblazer 2.1 instead of 2.0.
